listenbrainz_spark/recommendations/create_dataframes.py

```diff
@@ -152,27 +172,17 @@ def main():
     recordings_df_time = '{:.2f}'.format((time() - t0) / 60)
 
     current_app.logger.info('Preparing listen data dump and playcounts, saving playcounts to HDFS...')
-    t0 = time()
-    try:
-        listens_df = sql.prepare_listen_data(table)
```

Comment:
What happened to this function? Is it still in use? Not sure I like that all the logic has moved to the top level function...

Reply:
`listens_df = complete_listens_df.select('recording_mbid', 'user_name')`
This does the same thing as the above function. As we discussed, we will slowly deprecate the usage of queries and use API instead.  
What do you mean by top-level? And what specifically you don't like about it...